python/cuml/neighbors/kneighbors_classifier.pyx

```diff
@@ -71,27 +71,82 @@ cdef extern from "cuml/neighbors/knn.hpp" namespace "ML":
         int k
     ) except +
 
+"""
+K-Nearest Neighbors Classifier is an instance-based learning technique,
+that keeps training samples around for prediction, rather than trying
+to learn a generalizable set of model parameters.
+
+Examples
+---------
+.. code-block:: python
+
+  from cuml.neighbors import KNeighborsClassifier
+
+  from sklearn.datasets import make_blobs
+  from sklearn.model_selection import train_test_split
+
+  X, y = make_blobs(n_samples=100, centers=5,
+                    n_features=10)
+
+  knn = KNeighborsClassifier(n_neighbors=10)
+
+  X_train, X_test, y_train, y_test =
+    train_test_split(X, y, train_size=0.80)
+
+  knn.fit(X_train, y_train)
+
+  knn.predict(X_test)
+
+
+Output:
+
+
+.. code-block:: python
+
+  array([3, 1, 1, 0, 2, 0, 0, 0, 0, 0, 0, 0, 1, 2, 3, 1, 0, 0, 0, 2, 3, 3,
+         0, 3, 0, 0, 0, 0, 3, 2, 0, 0, 0], dtype=int32)
+
+
+Notes
+------
+
+For additional docs, see `scikitlearn's KNeighborsClassifier
+<https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html>`_.
+"""
+
+
 class KNeighborsClassifier(NearestNeighbors):
```

Comment:
I thought you were removing KNeighborsClassifier for now?

Reply:
`KNeighborsClassifier` and `KNeighborsRegressor` are ready to go. I removed what I had of the MNMG KNeighborsClassifier because it was incomplete and would just have added more complexity to this already gigantic PR.